definitions/npm/dompurify_v1.x.x/test_dompurify_v1.x.x.js

```diff
@@ -0,0 +1,65 @@
+// @flow
+import { describe, it } from 'flow-typed-test';
+import createDOMPurify from 'dompurify';
+
+const DOMPurify = createDOMPurify();
+const dirty = `
+<div>
+  <h1>
+    Hello, <a href="#" name="world">World</a>
+  </h1>
+  <img src="hello.png" alt="world" />
+</div>`;
+
+describe('#createDOMPurify', () => {
+  it('should add/remove hook', () => {
+    DOMPurify.addHook('beforeSanitizeElements', (currentNode, data, config) => {
+      return currentNode;
+    });
+
+    // $ExpectError
+    DOMPurify.addHook('invalidHookName', (currentNode, data, config) => {
+      return currentNode;
+    });
+
+    DOMPurify.removeHook('beforeSanitizeElements');
+    DOMPurify.removeHooks('beforeSanitizeElements');
+    DOMPurify.removeAllHooks();
+  });
+
+  it('should sanitize', () => {
+    const clean = DOMPurify.sanitize(dirty);
+    console.log(clean.toLowerCase());
```

Comment:
Please remove `console.log` statements from the test, we generally don't leave them in.

Reply:
Done